config/vufind/permissions.ini

```diff
@@ -76,6 +76,9 @@
 ; access.StaffViewTab - Controls access to the staff view tab in record mode
 ; access.SummonExtendedResults - Controls visibility of protected Summon results
 ; feature.Favorites - Controls access to the "save favorites" feature
+[access.AdminModule]
```

Comment:
This was a mistake.  how does one revert this?  I have tried several variations of this:
`git checkout origin/master config/vufind/permissions.ini`
If you can nuke this from the PR easily, go for it.

Reply:
In this case, since it's just some added lines, the easiest thing is probably to use the web-based editor and simply delete the offending content. I'll do that right now, which should take care of this issue. Thanks for the clarification!